app/services/usps_in_person_proofing/transliterable_validator.rb

```diff
@@ -32,7 +32,7 @@ def initialize(options)
     # @param [ActiveModel::Validations] record
     def validate(record)
       return unless IdentityConfig.store.usps_ipp_transliteration_enabled
-
+      nontransliterable_chars = []
```

Comment:
If we make this a set, we don't need to `uniq` before logging
```suggestion
      nontransliterable_chars = Set.new
```